bc_obps/reporting/models/rls_configs/facility_report.py

```diff
@@ -0,0 +1,34 @@
+from reporting.enums.enums import ReportingTableNames
+from rls.enums import RlsRoles, RlsOperations
+from rls.utils.helpers import generate_rls_grants, generate_m2m_rls
+
+
+class Rls:
+    role_grants_mapping = {
+        RlsRoles.INDUSTRY_USER: [
+            RlsOperations.SELECT,
+            RlsOperations.INSERT,
+            RlsOperations.UPDATE,
+            RlsOperations.DELETE,
```

Comment:
Another comment for future refactoring, a meta-operation `RlsOperations.ALL` for readability and simplicity's sake. Pretty unlikely that we would add an element in the CRUD operations 😄